projects/snapshot_bot/snapshot_bot.cc

```diff
@@ -459,6 +463,9 @@ class RobotFSM : FSM<State>::StateHandler
         // Also log Vicon frame number, if present
         fieldNames.emplace_back("Frame");
 
+        // Also log the imu data
+        fieldNames.emplace_back("IMU [°]");
```

Comment:
This needs to be broken into three separate fields (see above). I'd go with something like ``IMU yaw [degrees]..`` etc. without the unicode.